src/sql/com/example/components/parser.clj

```diff
@@ -16,6 +16,7 @@
     [com.example.model.timezone :as timezone]
     [com.fulcrologic.rad.attributes :as rad.attr]
     [com.example.model.invoice :as invoice]
+    [com.wsscode.pathom.core :as p]
```

Comment:
could you explain why we need this?

Reply:
Oh, I see. The addition of transduce maps failed to require it...how did *that* ever work?